authorization/v1/generated.pb.go

```diff
@@ -7297,7 +7297,7 @@ func init() {
 }
 
 var fileDescriptorGenerated = []byte{
-	// 1821 bytes of a gzipped FileDescriptorProto
+	// 1820 bytes of a gzipped FileDescriptorProto
```

Comment:
this is surprising. Do you have the right protoc version?

Reply:
that was exactly the issue.

Reply:
I had 3.6.1 locally, and `update-protobuf.sh` does not fail on wrong versions, so it slipped through.